plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx

```diff
@@ -338,6 +341,7 @@ export const timeSeriesSection = [
         'of query results',
     ),
     controlSetRows: [
+      // eslint-disable-next-line react/jsx-key
```

Comment:
Can move this to top of file to disable the rule for the whole file.

Reply:
I'm wary of disabling rules for a whole file, it's safer to do it per-line.

Reply:
For this particular case, I don't understand why the rule needs to be disabled though. I see no real code change and it certainly has passed previous CI check?

Reply:
previously it was in the `temporary-plugins` dir which is excluded from CI build and tests